be/src/exec/vectorized/aggregate/aggregate_streaming_node.cpp

```diff
@@ -231,8 +232,10 @@ std::vector<std::shared_ptr<pipeline::OperatorFactory> > AggregateStreamingNode:
     context->add_pipeline(operators_with_sink);
 
     OpFactories operators_with_source;
-    operators_with_source.emplace_back(
-            std::make_shared<AggregateStreamingSourceOperatorFactory>(context->next_operator_id(), id(), aggregator));
+    auto source_operator =
+            std::make_shared<AggregateStreamingSourceOperatorFactory>(context->next_operator_id(), id(), aggregator);
+    source_operator->set_num_driver_instances(1);
```

Comment:
Why set all source operator instance  number  to  1 ?

Reply:
Currently, the shared data structure aggregator does not support concurrency.
I'll add TODO items, and fix it later